regression-test/README.md

```diff
@@ -24,18 +16,18 @@ In order to run the test you have to have it's dependencies installed. (this pro
 npm install
 ```
 
-Then build the project(with `esbuild`):
+Then build the project:
 
 ```sh
 # cwd: regression-test
-npm run build:esbuild
+npm run build
 ```
 
-The regression testing tool (`testcafe`) will test the app in a real browser, so we have to have a server serving our application:
+The regression testing tool will test the app in a real browser, so we have to have a server serving our application:
```

Comment:
I would reword this saying it is starting the next server, also you don't need `run` before `start` so it's just `npm start`

Reply:
Good point.